scripts/deploy

```diff
@@ -0,0 +1,189 @@
+hn096OJKnGZFb/HiW9iJWhj4CJ4LubSvsiZJZ7YuDlM9faOBsjCBrzAOBgNVHQ8B
+Af8EBAMCAqQwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDwYDVR0TAQH/BAUwAwEB/zAd
+BgNVHQ4EFgQUImjresql78fBpwSV7lp4fT4+NnwwWAYDVR0RBFEwT4IFbWluaW+C
+C21pbmlvLm1pbmlvgg9taW5pby5taW5pby5zdmOCHW1pbmlvLm1pbmlvLnN2Yy5j
+bHVzdGVyLmxvY2Fsgglsb2NhbGhvc3QwCgYIKoZIzj0EAwIDSAAwRQIgWT4CU5ib
+LNeXJmh2lnqEvaeKgqLHPFgMOQg+4TyO+uQCIQCI5WX1E84B+z6yX7WKIBYJIjto
+RjQi75QniF10pi2jKA==
+-----END CERTIFICATE-----
+EOF
+
+  cat <<EOF > private.key
+-----BEGIN PRIVATE KEY-----
+MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgRiROiBUTvzxfDbiu
+60w9WdlRKAI7Jy0c26uC2FoAZdahRANCAAQUfxQ8KX+8AeQxtMXcTzKFJ0s0fZe5
+4pQeez47gVGGfT3o4kqcZkVv8eJb2IlaGPgIngu5tK+yJklnti4OUz19
+-----END PRIVATE KEY-----
+EOF
+}
+
+deploy_minio() {
+  if [[ -z "${KUBECONFIG}" ]]; then
+   echo "Please set your Kubeconfig environment variable before running."
+   exit 0
+  fi
+
+  helm repo add minio https://charts.min.io/
+  helm repo update
+
+  echo "Creating Minio Namespace"
+  kubectl create ns minio
+
+  echo "Creating Minio tls-ssl secret"
+  kubectl -n minio create secret generic tls-ssl-minio --from-file=private.key --from-file=public.crt
+
+  echo "Installing Minio"
+  helm install --namespace minio --set rootUser=inspectorgadget,rootPassword=gogadgetgo --set tls.enabled=true,tls.certSecret=tls-ssl-minio --set replicas=1 --set resources.requests.memory=2Gi --set persistence.enabled=false --set drivesPerNode=0 --set pools=0 --set mode=standalone --set buckets[0].name=rancherbackups --set buckets[0].policy=none --set buckets[0].purge=true minio minio/minio
+
+  kubectl create secret generic miniocreds --from-literal=accessKey=inspectorgadget --from-literal=secretKey=gogadgetgo
+}
+
+k3s_deploy_minio() {
+  helm repo add minio https://charts.min.io/
+  helm repo update
+
+  echo "Creating Minio Namespace"
+  k3s kubectl create ns minio
+
+  echo "Creating Minio tls-ssl secret"
+  k3s kubectl -n minio create secret generic tls-ssl-minio --from-file=private.key --from-file=public.crt
+
+  echo "Installing Minio"
+  helm install --namespace minio --set rootUser=inspectorgadget,rootPassword=gogadgetgo --set tls.enabled=true,tls.certSecret=tls-ssl-minio --set replicas=1 --set resources.requests.memory=2Gi --set persistence.enabled=false --set drivesPerNode=0 --set pools=0 --set mode=standalone --set buckets[0].name=rancherbackups --set buckets[0].policy=none --set buckets[0].purge=true minio minio/minio
+
+  time timeout 300 bash -c 'while ! (k3s kubectl --namespace minio rollout status --timeout 10s deploy/minio 2>/dev/null); do sleep 5; done'
+
+  echo "Creating Minio secret"
+  k3s kubectl create secret generic miniocreds --from-literal=accessKey=inspectorgadget --from-literal=secretKey=gogadgetgo
+}
+
+deploy_backup_restore() {
+  export TAG=test
+  if [[ -z "${KUBECONFIG}" ]]; then
+   echo "Please set your Kubeconfig environment variable before running."
+   exit 0
+  fi
+  if [[ -z "${DOCKERHUB_USER}" ]]; then
```

Comment:
Now that I see this, I thought we concluded on not having the requirement on a DockerHub account and going with the way the current integration test does it (exporting and importing the image).

This won't work for k3s as there is a dependency on SSH but should work with k3d.